crates/subspace-networking/src/node_runner.rs

```diff
@@ -339,8 +339,6 @@ where
             }
         };
 
-        debug!("Bootstrap started.");
```

Comment:
Why do we want to remove this?

Reply:
We already print exactly the same message above on line 315, so we print it twice for no reason. Probably artifact of some refactoring.